File: Programmers/Lv.1/문자열다루기기본.py
def solution(s):
    answer = True
    
    if len(s) == 4 or len(s) == 6:
        answer = True
    else:
        answer = False
    if answer == True:
        if s.isdigit():
            answer = True
        else:
            answer = False
    
    return answer

# def solution(s):
#     return s.isdigit() and len(s) in (4, 6)

File: Programmers/Lv.1.Kotlin/문자열다루기기본.kt
class Solution {
    fun solution(s: String): Boolean {
        return s.all {
            it.isDigit()
        } && ((s.length == 4 || s.length == 6))

        // val length = s.filter { it.isDigit() }.length
        // return (length == 4 || length == 6) && length == s.length
    }
}